### DISCUSSION TITLE
Menu组件如何在后端返回数据时做到hide菜单的父菜单项高亮

### DISCUSSION THREAD

**Discussion by lppoi** [ant-design/ant-design#44493] [Q&A]
目前我的路由是
/user 用户管理
-----/user/add 新增用户
-----/user/edit/:id 编辑用户
/test
-----/test/list
-----/test/list/add
-----/test/list/edit/:id
通过Menu渲染时我会隐藏/add、/edit路由，我希望跳转该页面后应该高亮用户管理菜单。

还望各位大佬可以指点一下。

**Comment by soonoosoon** [comment]
通过 Menu 的 `defaultOpenKeys` 属性可以高亮父节点
```
<Menu defaultOpenKeys={[defaultOpenKey]} /> 
```
